.github/workflows/tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Python tests
+
+on:
+  push:
+    branches: 'master'
+  pull_request:
+    branches: 'master'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
Also, while you're on it, please integrate yamllint with the following configuration:
* https://github.com/sphinx-contrib/sphinxcontrib-towncrier/blob/master/.yamllint
* https://github.com/sphinx-contrib/sphinxcontrib-towncrier/blob/7588aed/.pre-commit-config.yaml#L119-L126

This will help you keep the style in the YAML files consistent. You can do this in a separate PR but an atomic commit in this one would be fine as well.

Reply:
fine, i try to do it